crates/biome_html_formatter/src/html/lists/attribute_list.rs

```diff
@@ -6,22 +6,32 @@ pub(crate) struct FormatHtmlAttributeList;
 impl FormatRule<HtmlAttributeList> for FormatHtmlAttributeList {
     type Context = HtmlFormatContext;
     fn fmt(&self, node: &HtmlAttributeList, f: &mut HtmlFormatter) -> FormatResult<()> {
-        let attribute_len = node.iter().len();
-        let line_break = if f.options().attribute_position() == AttributePosition::Multiline
-            && attribute_len > 1
+        let attribute_count = node.iter().len();
```

Comment:
We can use `node.len()`, which we already use a few lines below. We can use the same variable